src/entities/SecurityToken/Permissions.ts

```diff
@@ -6,16 +6,19 @@ import { AssignSecurityTokenRole } from '../../procedures';
 import { SecurityTokenRole, Feature, ErrorCode } from '../../types';
 import { PolymathError } from '../../PolymathError';
 
+/**
+ * Permissions implementation used to manage the permissions flags that are associated with STO
+ */
 export class Permissions extends SubModule {
   /**
    * Get a list of all available roles.
    * The returned roles depend on which features are enabled, as per:
    *
-   * Feature           Roles
-   * -------------------------
-   * Permissions       Permissions Administrator
-   * Shareholders      Shareholders Administrator
-   * Dividends         Dividends Administrator, Dividends Operator
+   * | Feature      | Roles                                       |
+   * |--------------|---------------------------------------------|
+   * | Permissions  | Permissions Administrator                   |
+   * | Shareholders | Shareholders Administrator                  |
+   * | Dividends    | Dividends Administrator, Dividends Operator |
```

Comment:
This table should be updated with the restrictions features and roles (ShareholderCountRestrictions, PercentageOwnershipRestrictions and their respective roles)